javacord-api/src/main/java/org/javacord/api/event/message/reaction/ReactionAddEvent.java

```diff
@@ -14,4 +14,11 @@ public interface ReactionAddEvent extends SingleReactionEvent {
      */
     CompletableFuture<Void> removeReaction();
 
+    /**
+     * Gets the message author id.
+     *
+     * @return The message author id.
+     */
+    long getMessageAuthorId();
```

Comment:
```suggestion
    Optional<Long> getMessageAuthorId();
```
This is an optional field and should return an `Optional<Long>`

Reply:
Good point, thank you!